src/components/notifications/ui-notification-toast.tsx

```diff
@@ -123,7 +100,12 @@ export const UiNotificationToast: React.FC<UiNotificationProps> = ({
         <small>{date.toRelative({ style: 'short' })}</small>
       </Toast.Header>
       <Toast.Body>{contentDom}</Toast.Body>
-      <ProgressBar variant={'info'} now={eta} max={durationInSecond * STEPS_PER_SECOND} min={0} />
+      <ProgressBar
+        variant={'info'}
+        now={remainingSteps}
+        max={durationInSecond * STEPS_PER_SECOND}
```

Comment:
Should `durationInSecond * STEPS_PER_SECOND` be wrapped inside a `useMemo` or is this not necessary here?

Reply:
The result is a number which is a primitive datatype that is easy to compare. useMemo is only necessary if you have complex data types like dicts, class instances or functions that would get recreated on every render.